### DISCUSSION TITLE
Repositório em java, mas aparece como CSS

### DISCUSSION THREAD

**Discussion by gabrielolvrz** [frontendbr/forum#2485] [Perguntas e Respostas]
Estou criando um repositório no GitHub para armazenar exercícios da minha disciplina de Linguagem de Programação Orientada a Objetos (LPOO) em Java. Incluí imagens .png das perguntas nos exercícios, mas o GitHub as reconhece como arquivos CSS. Alguém tem alguma dica de como posso corrigir isso ou trocar o tipo de arquivo?

**Comment by igorlvicente** [comment]
Não há problema nenhum nisso. É só uma funcionalidade visual "legal" que o Github oferece. Se tudo estiver certo (os arquivos estão corretamente adicionados ao repositório), só ignore.
